common/changes/master_2017-04-25-02-07.json

```diff
@@ -0,0 +1,25 @@
+{
+  "changes": [
```

Comment:
would be great if you could add an example using this prop. It will make it easy to test. Thanks